tests/utils/matrix_utils.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
```

Comment:
Factored this out as importing Eigen with WAMR broke WAMR imports (they both seem to define a `Vector` class)